test/blackbox_tests/outbound_redirect_test.go

```diff
@@ -282,6 +284,118 @@ var _ = Describe("Outbound IPv4 TCP traffic to any address:port except excluded
 	)
 })
 
+var _ = Describe("Outbound IPv4 TCP traffic to any address:port except ports excluded by uid ones", func() {
```

Comment:
In this test I call to the `excludedPort` twice, once using a normal user and then using a `4201` uid, once I assert that I get the redirected response and once `excluded`. The mirror test is for ipv6.